streams/src/test/java/org/apache/kafka/test/InternalMockProcessorContext.java

```diff
@@ -369,6 +370,16 @@ public long timestamp() {
         return recordContext.timestamp();
     }
 
+    @Override
+    public long currentSystemTimeMs() {
+        return Time.SYSTEM.milliseconds();
```

Comment:
We should pass in a `Time` parameter into the constructor of `InternalMockProcessorContext` and return `time.milliseconds()` here so we can mock the time.

If this method is unused, it's also ok to just throw an exception for now and we add mocking capabilities later when needed.

(Same below.)